modules/mergeImageWithTelemetry/mergeImageWithTelemetryWorker.py

```diff
@@ -0,0 +1,39 @@
+from modules.mergeImageWithTelemetry.mergeImageWithTelemetry import MergeImageWithTelemetry
+
+def pipelineMergeWorker(pause, exitRequest, imagePipelineIn, telemetryPipelineIn, pipelineOut):
+    print("start pipeline merge")
+    mergeImageWithTelemetry = MergeImageWithTelemetry()
+    curImage = None
+
+    while True: 
+        if not exitRequest.empty():
+            print("end pipeline merge")
+            return
+        
+        pause.acquire()
+        pause.release()
+
+        # put current elements of the telemetry pipeline into the queue
+        telemetryPipelineIn.acquire()
+        while True: 
+            try: 
+                telemetryData = telemetryPipelineIn.get_noawait()
+                mergeImageWithTelemetry.put_back_telemetry(telemetryData)
+            except: 
+                break
+        telemetryPipelineIn.release()
+        
+        if mergeImageWithTelemetry.should_get_image():
+            try: 
+                curImage = imagePipelineIn.get_noawait()
```

Comment:
There's no need to constantly try to get an image, if there's no image right now it's fine to wait patiently for one.

Basically right now the worker is going to be spinning a lot, which is wasteful.

Reply:
So something like:
```
if mergeImageWithTelemetry.should_get_image():
    curImage = imagePipelineIn.get()  # We can wait for an image, no rush
```
is fine.